tests/acceptance/pageObjects/filesPage.js

```diff
@@ -257,10 +257,12 @@ module.exports = {
       })
     },
     confirmFileOverwrite: function () {
-      return this
-        .waitForElementVisible('@fileOverwriteConfirm')
-        .click('@fileOverwriteConfirm')
-        .waitForElementNotVisible('@fileOverwriteConfirm')
+      return this.waitForElementVisible('@fileOverwriteConfirm')
+        .api.moveToElement(this.elements.fileOverwriteDialog.selector, 0, 0)
+        .mouseButtonClick(0)
+        .waitForElementNotVisible(this.elements.newFolderButton.selector)
```

Comment:
This is important because we upload the file directly using `setValue`. The file upload dropdown does not get closed here (the overwrite dialog is too fast that it does not give time to close the dropdown). So, what this does is, the `fileOverwriteDialog` has an overlay, and we click that using `mouse`, so as to remove the dropdown. The code follows what was before afterward.

Reply:
Seems OK. But please add some comment on the code explaining what the code is doing. That way we will understand this when we look it in the future.